README.md

```diff
@@ -112,7 +101,12 @@ The type of `formatted` will depend on the formatter function and can be safely
 
 Three formatters are included out of the box:
 
-* `SQLDatabaseFormatter` returns format to SQL technology such as PostgreSQL and MySQL.
+* `SQLDatabase` returns format to SQL technology such as PostgreSQL and MySQL.
+* `MongoDB` returns a MongoDB connection as MongoClient instance
```

Comment:
Missing period at the end. Also, this implies it returns an instantiated MongoClient connection object, rather than just a formatted string.  That's more than it's supposed to do.